cmd/minikube/cmd/start.go

```diff
@@ -574,6 +570,11 @@ func bootstrapCluster(bs bootstrapper.Bootstrapper, r cruntime.Manager, runner b
 			exit.WithLogEntries("Error restarting cluster", err, logs.FindProblems(r, bs, runner))
 		}
 		return
+	} else {
+		console.OutStyle("pulling", "Pulling images required by Kubernetes %s ...", kc.KubernetesVersion)
```

Comment:
I like this change - but there is a bit more subtlety here that is missing. This will also need to handle the case where the pre-existing cluster is a different version of Kubernetes.

Reply:
Here in Brazil it really saves me 30 seconds. So I will later send you will a PR to check if the server is being upgraded or not.

I read that you aim to make minikube launch in less than a minute. 

With this patch (and all the others you have merged), here it loads in 53 seconds. How long does it take to load for you ?